eclipse/tests/org.switchyard.tools.m2e.tests/src/org/switchyard/tools/m2e/tests/AbstractSwitchYardTest.java

```diff
@@ -32,7 +35,7 @@
      * 
      * @throws Exception
      */
-    protected void waitForJobs() throws Exception {
+    protected void waitForJobsOld() throws Exception {
```

Comment:
is it still used?
seems not we might remove it or at least set @deprecated annotation